docker-compose.yml

```diff
@@ -154,3 +154,12 @@ services:
     image: redis:4.0.14-alpine
     ports:
       - 4001:6379
+
+  redis-commander:
+    image: rediscommander/redis-commander:latest
+    depends_on:
```

Comment:
TIL you no longer needs `links` as well - always used to.

Which is nice

Reply:
Ta. I'll push this through and clean up on my next PR.